src/nbl/video/IOpenGL_PhysicalDeviceBase.h

```diff
@@ -19,15 +19,19 @@
 namespace nbl::video
 {
 
+class IOpenGLPhysicalDeviceBase
+{
+public:
+	static inline constexpr uint32_t MaxQueues = 8u;
+};
+
 template <typename LogicalDeviceType>
-class IOpenGL_PhysicalDeviceBase : public IPhysicalDevice
+class IOpenGL_PhysicalDeviceBase : public IPhysicalDevice, public IOpenGLPhysicalDeviceBase
```

Comment:
let `IOpenGLPhysicalDeviceBase` do the inheritance from `IPhysicalDevice`

Reply:
then I would need a passthrough constructor.